java-manta-client/src/test/java/com/joyent/manta/client/crypto/EncryptingEntityTest.java

```diff
@@ -74,9 +74,10 @@ public void canEncryptAndDecryptToAndFromFileInAesCbc() throws Exception {
         verifyEncryptionWorksRoundTrip(AesCbcCipherDetails.INSTANCE_128_BIT);
     }
 
-    public void canEncryptAndDecryptToAndFromFileWithManySizesInAesCbc() throws Exception {
-        canEncryptAndDecryptToAndFromFileWithManySizes(AesCbcCipherDetails.INSTANCE_128_BIT);
-    }
+    // FIXME
```

Comment:
The style checker ignores test classes because I have found it to be a huge productivity drain. In the future, we could do a more lenient check on test classes.